Src/Support/Google.Apis.Auth.AspNetCore/GoogleOpenIdConnectExtensions.cs

```diff
@@ -0,0 +1,135 @@
+﻿/*
+Copyright 2018 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Auth.AspNetCore;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OpenIdConnect;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using System;
+using System.Linq;
+
+// Microsoft recommend that all service Add methods go in this namespace.
+// See https://docs.microsoft.com/en-us/aspnet/core/fundamentals/dependency-injection
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class GoogleOpenIdConnectExtensions
```

Comment:
More XML comments please

Reply:
Funnily enough, none of the MS equivalent classes have nay XmlDoc! E.g. https://github.com/aspnet/Security/blob/master/src/Microsoft.AspNetCore.Authentication.Facebook/FacebookExtensions.cs

I've added some here regardless.